tests/python/mkl/subgraphs/test_conv_subgraph.py

```diff
@@ -0,0 +1,723 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import copy
+import mxnet as mx
+import pytest
+from subgraph_common import check_fusion, check_neg_fusion, check_quantize
+from subgraph_common import CustomNormalInit, DATA_SHAPE, RELU6, TailNegBlock
+from subgraph_common import DATA_SHAPE, SG_PASS_NAME, QUANTIZE_SG_PASS_NAME
+from mxnet.contrib import quantization
+from mxnet.gluon import nn
+from mxnet.test_utils import assert_almost_equal, assert_almost_equal_with_err
+
+def test_float64_fallback():
```

Comment:
with_seed annotation

Reply:
we still have that, right @szha @leezu ?

Reply:
It's no longer needed to annotate every test function with pytest, but rather all test functions are automatically annotated. You can see https://github.com/apache/incubator-mxnet/commit/3f436fb1fe1a839f6f5d981e3ec626cd840045df

Reply:
Ah right, thanks!